primitives/messages/src/target_chain.rs

```diff
@@ -97,6 +97,7 @@ pub trait MessageDispatch<AccountId, Fee> {
 	/// of dispatch weight.
 	fn dispatch_weight(message: &DispatchMessage<Self::DispatchPayload, Fee>) -> Weight;
 
+	fn pre_dispatch(message: &DispatchMessage<Self::DispatchPayload, Fee>) -> bool;
```

Comment:
```suggestion
	fn check_receiving(message: &DispatchMessage<Self::DispatchPayload, Fee>) -> bool;
```

Reply:
Current naming looks good to me, although `receiving` helps me better to understand this step.

Here `pre_dispatch` is not part of `dispatch`

There are lots of other `pre_dispatch` in other traits may different meaning.

Reply:
Yeah. the `receiving` related name is too specific IMO. We are building a library, given a more general name is better.